### PR TITLE
Thirdeye add onboard endpoints in dashboard resources for anomaly function clone

### PR DESCRIPTION
**Pull request by bl44**
Create a new rest endpoints in dashboard resource
1. provide a method that can clone anomaly function 1 by1 or in batch
function name of new cloned function is [original_name] + '_' + [tag] where [tag] is provided in the query

this functionality has been tested against local database and dashboard appliation

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/OnboardResource.java [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+            nameTagList.add("");
+          }
+        }
+      }
+    }
+
+    LOG.info("function ids set: {}", functionIds);
+    LOG.info("name tag set: {}", nameTagList);
+    // Construct Return results
+    HashMap<Long, String> IdNameTagMap = new HashMap<>();
```
Comment (cyy0714) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/OnboardResource.java]: variable name should start with lowercase letter.

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/OnboardResource.java [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+    AnomalyFunctionDTO anomalyFunction = anomalyFunctionDAO.findById(id);
+    // if cannot find then return
+    if (anomalyFunction == null) {
+      // LOG and exit
+      LOG.error("Anomaly Function With id [{}] does not found", id);
+      return null;
+    }
+    String functionName = anomalyFunction.getFunctionName();
+    String newFunctionName = functionName;
+    if (cloneNameTag != null && cloneNameTag.length() > 0) {
```
Comment (cyy0714) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/OnboardResource.java]: would it be better to use cloneNameTag.isEmpty()?

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/OnboardResource.java (hunk 1 above)
Comment (bl44) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/OnboardResource.java]: sorry for not address this comments before merge, will do it next time when working on similar changes.

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/OnboardResource.java (hunk 2 above)
Comment (bl44) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/OnboardResource.java]: Thanks for this. Lesson learned.

**Pull request merged**